AmplifyPlugins/DataStore/Sources/AWSDataStorePlugin/Storage/CascadeDeleteOperation.swift

```diff
@@ -465,10 +505,23 @@ public class CascadeDeleteOperation<M: Model>: AsynchronousOperation {
                                     completion: @escaping DataStoreCallback<MutationEvent>) {
         syncEngine.submit(mutationEvent, completion: completion)
     }
+
+    private func _log(
```

Comment:
the `_` prefix on function name looks like a `C++` program style.

Reply:
Agreed, renamed to `logMessage`